src/components/entity/ha-entity-picker.ts

```diff
@@ -260,6 +264,19 @@ export class HaEntityPicker extends LitElement {
     }
   }
 
+  private _filterChanged(ev): void {
+    const filterString = ev.detail.value.toLowerCase();
```

Comment:
Should `fuzzySequentialMatch` take care of the lower case?

Reply:
I've gone back and forth on that. I eventually landed that it made more sense for the client to decide whether or not casing should matter.

But that being said, I can't really think of any situations in Home Assistant where you'd want to use case-sensitive filtering.

I'd be fine with moving it into the matcher.

Reply:
I think it would make sense to move it to the matcher. We could add a flag eventually to match case... But it would make for less duplicate code and simpler implementation.